src/components/menu/menu-test.stories.tsx

```diff
@@ -28,6 +28,8 @@ const meta: Meta<typeof Menu> = {
     "InNavigationBarStory",
     "MenuFullScreenKeysTest",
     "MenuWithTwoSegments",
+    "MenuFullScreenWithLargeMenuItems",
+    "MenuComponentFullScreenWithLongSubmenuText",
```

Comment:
**suggestion**: these stories don't appear to be defined in this file. Happy for us to include the long-text MenuItems in the main test story though 👍🏼

```suggestion
```